src/applications/debt-letters/const/diary-codes/index.js

```diff
@@ -81,6 +85,21 @@ export const diaryCodes = Object.freeze({
   '820': 'Your waiver request is being reviewed',
 });
 
+const ContactDMC = () => (
+  <span>
+    {CONTACTS.DMC ? <Telephone contact={CONTACTS.DMC} /> : '800-827-0648'} (or{' '}
+    {CONTACTS.DMC_OVERSEAS ? (
+      <Telephone
+        contact={CONTACTS.DMC_OVERSEAS}
+        pattern={PATTERNS.OUTSIDE_US}
+      />
+    ) : (
+      '1-612-713-6415'
+    )}{' '}
```

Comment:
Would this work inside the a telephone component?

```suggestion
    <Telephone
      contact={CONTACTS.DMC_OVERSEAS || '1-612-713-6415'}
      pattern={PATTERNS.OUTSIDE_US}
    />
    {' '}
```